### PR TITLE
VPN Updates

### PR DESCRIPTION
**Pull request by pri-lk**
┆Issue is synchronized with this [Wrike task](https://www.wrike.com/open.htm?id=1269722964)

### CODE REVIEW

**Review by pallabpain — CHANGES_REQUESTED** [review]

File: hugo/content/3_how-tos/34_networking-and-communication/347_enable-vpn.md [hunk 1]
```diff
@@ -103,29 +103,63 @@ To list projects with features:
 rio project list -w
 ```
 
-To create a new project you will have to first create a project maifest, set the attribute `vpn.enabled` to `true` and then apply the manifest.
+To create a new project you will have to first create a project maifest, set the attribute `vpn.enabled` to `true` and then apply the manifest.
```
Comment (pallabpain) [inline hugo/content/3_how-tos/34_networking-and-communication/347_enable-vpn.md:106]:
```suggestion
To create a new project you will have to first create a project manifest, set the attribute `vpn.enabled` to `true` and then apply the manifest. 
```

File: hugo/content/3_how-tos/34_networking-and-communication/347_enable-vpn.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ To enable or disable VPN for an existing project, follow these steps:
 
   1. Navigate to the right navigation pane and click **Devices**.
   2. Select the specific device for which you want to enable VPN.
-  3. On the device details page, click **Enabled** next to the VPN option. For more information, see [Enable VPN for an Online Device](/3_how-tos/32_device-management/321_onboarding-a-device/#enable-vpn-for-an-online-device)
+  3. On the device details page, click **Enable VPN**.  A pop-up will appear with the **Advertise Routes** field. To advertise your device on the subnets specified during project creation, mark the checkbox. For more information, see [Enable VPN for an Online Device](/3_how-tos/32_device-management/321_onboarding-a-device/#enable-vpn-for-an-online-device)
```
Comment (pallabpain) [inline hugo/content/3_how-tos/34_networking-and-communication/347_enable-vpn.md:72]: @SanchithaSridhar Does the pop-up appear for all cases or only when the project has subnets?

File: hugo/content/3_how-tos/31_account-management/311_managing-projects.md [hunk 3]
```diff
@@ -43,9 +43,10 @@ Managing projects include creating and deleting projects as well as adding and r
        Only an org admin or a project creator can enable/disable VPN on projects. <br>
        When you create your first project, you cannot view the VPN option. However, you can enable it after creating the project.<br>
        For more information, see [Enabling VPN Services](/3_how-tos/34_networking-and-communication/347_enable-vpn/)
-    4. (Optional) Select the **Users** tab to add user to the project.
-    5. (Optional) Select the **User groups** tab to add user groups to the project.
-    6. Click **Create Project**. The newly created project can be viewed in the project dashboard.
+    4. Enter the values for the subnets. This option is displayed only when VPN is enabled. You can input a single value or multiple values as a comma-separated list. This field is optional. If multiple values are provided, they will be advertised on all the routes.
+    5. (Optional) Select the **Users** tab to add user to the project.
+    6. (Optional) Select the **User groups** tab to add user groups to the project.
+    7. Click **Create Project**. The newly created project can be viewed in the project dashboard.
```
[the remark this answers is not in the document]
Comment (pri-lk) [reply hugo/content/3_how-tos/31_account-management/311_managing-projects.md:49]: we do not add a lot of screenshots in developer docs, and also its pretty straight forward.

File: hugo/content/3_how-tos/34_networking-and-communication/347_enable-vpn.md (hunk 2 above)
Comment (pri-lk) [reply hugo/content/3_how-tos/34_networking-and-communication/347_enable-vpn.md]: yes it appears only if the projects have subnets, have updated the content

**Review by pallabpain — APPROVED** [review]

**Pull request merged**